client/package.json

```diff
@@ -73,6 +75,7 @@
     "react-popper": "^2.3.0",
     "react-redux": "^8.1.2",
     "react-router-dom": "^5.2.0",
+    "react-router-dom-v5-compat": "^6.26.0",
```

Comment:
what are the new react router dom apis being used?

Reply:
This PR is using the new API for `NavLink`, which lets you pass `isActive` to the children of the component, see: https://reactrouter.com/en/main/components/nav-link#children This isn't available in react router dom v5, I'd have to write more CSS selectors and this approach felt like it was cleaner when used alongside Radix UI.

Installing `react-router-dom-v5-compat` alongside `react-router-dom` is the recommended approach for migrating from v5 to v6 because it allows us to incrementally update the code.